### PR TITLE
Update to support type checking

### PR DESCRIPTION
**Pull request by cakekoa**

### CODE REVIEW

**Review by mssalvatore — CHANGES_REQUESTED** [review]
Missing changelog entry.  Here's what I have in toolbox:
image

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ the [PEP 440 version scheme](https://peps.python.org/pep-0440/#version-scheme).
 ## [v0.3.0 - 2023-11-27]
 ### Added
 - A specific PTH tag (T1550.002)
+- Support for type checking (py.typed marker file)
```
Comment (mssalvatore) [inline CHANGELOG.md:14]: This was added to v0.3.0, but it isn't in v0.3.0. it's unreleased ATM and will go into v0.4.0 (or v1.0.0 if we decide it's production ready)

Comment (mssalvatore) [inline CHANGELOG.md:14]: This was added to v0.3.0, but it isn't in v0.3.0. it's unreleased ATM and will go into v0.4.0 (or v1.0.0 if we decide it's production ready)

Comment (cakekoa) [reply CHANGELOG.md]: Whoops -- moved it to the Unreleased section

**Review by mssalvatore — APPROVED** [review]

**Pull request merged**